composer.json

```diff
@@ -15,6 +15,27 @@
 		"guzzlehttp/promises": "~1.0",
 		"psr/log": "~1.0"
 	},
+	"require-dev": {
+		"phpunit/phpunit": "~4.8.0|~5.3.0",
+		"jakub-onderka/php-parallel-lint": "0.9.2"
+	},
+	"suggest": {
+		"etsy/phan": "Allows running static analysis on the package (requires PHP 7+)"
```

Comment:
> It's a shame there isn't a suggest-dev.

Yeh, that would be ideal ;)
I'll knock something up to make phan run in CI for php7 etc at some point too!